### DISCUSSION TITLE
[324160] CLANNAD团子大家族

### DISCUSSION THREAD

**Discussion by Leslie159357** [pjy612/SteamManifestCache#360] [Share]
> appid:`324160` keys:
> 
> ```
>324161;cceaefea70200c5d9e6200661064d3f4bde2894727f071d4b5fa36f31ddcad2b
>324163;283cd6f1bfc6a44cfab2d56951b826b2f89b2c99091dbeb76f50f1ddbddc9c8c
>519110;2706a2712987f5edf16fd8293508ab25151efc5f09ca6a0180344280a0342b58
>615930;7569614c34e7f7b946e6d61307a64d92167b333e784f4eaf6cc976a6e3ee698d
> ```
> 
> CODE:`803F4434A5AE1DBE` EMAIL:[bai136699@163.com]
[324160.zip](https://github.com/pjy612/SteamManifestCache/files/13891305/324160.zip)

**Comment by Paperplaneyu** [comment]
清单有问题

**Reply by pjy612** [reply]
挖坟？造谣？
